lib/etl/extract/process_files_in_s3_bucket_folders.rb

```diff
@@ -0,0 +1,52 @@
+require 'aws-sdk'
+require 'tempfile'
+
+class ProcessFilesInS3BucketFolders
+  def initialize(bucket_name, prefix_folders, file_ext)
+    @bucket_name = bucket_name
+    @prefix_folders = prefix_folders
```

Comment:
Out of curiosity, why is this an array? It seems like you're only passing arrays with one value in the lower areas, so wouldn't it simplify this class to keep it as an individual folder vs. multiple folders?